src/user-research/getting-informed-consent-for-user-research.md

```diff
@@ -7,11 +7,29 @@ You must have informed consent from participants before you start any research a
 
 This guidance explains how we collect and keep evidence of informed consent at dxw. It builds on the general guidance on [getting informed consent](https://www.gov.uk/service-manual/user-research/getting-users-consent-for-research) in the GOV.UK Service Manual.
 
-## Using dxw or client consent materials and processes
+Note that this guidance applies to all dxw staff, dxw friends, and partner and client staff doing research activities, like interviews, visits, surveys and tests.
```

Comment:
This sentence refers to any type of research, as in business and technical research also or only user research? From the moment i started at dxw it was evident in my team that the business analyst does research. Should this sentence then be more specific for starters like me?

Reply:
The guidance applies to business analysts, or anyone else who is doing the kinds of research it describes.